### ISSUE TITLE
Problems with Tabulator 3.5.3

### ISSUE THREAD

**Issue by point-nine** [olifolkerd/tabulator#1503]
**Describe the bug**
I actually have 2 issues that **possibly** are bugs:

1) Column resizing is unpredictable ... sometimes it works, sometimes snaps back to original width.  On an iPad (specifically), it's unbearably slow to redraw when resizing does work correctly.

2) Despite setting 'resizableRows' to false, we are still getting resize handles and can resize the rows.

**Tabulator Info**
- Which version of Tabulator are you using?
3.5.3

- Post a copy of your construct object if possible so we can see how your table is setup

`$( '#table' ).tabulator({    
    autoResize: true
    , tooltipsHeader: true 
    , layout: "fitColumns"
    , placeholder: '... Loading ...'
    , resizableRows: false
    , resizableColumns: true <!---
    , virtualDom: true
    , layoutColumnsOnNewData: false
    , persistentLayout: true`

**To Reproduce**
1. Go to 'column heading'
2. Click on 'resizable handle'
3. Resize (larger/smaller)
4. May or may not resize to desired with, may snap back to original width

**Expected behavior**
The column to be resized

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Chrome
 - Version: Version 69.0.3497.100 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
 - Device: iPad
 - OS: iOS (unsure of version)
 - Browser: Chrome
 - Version Unsure of version

**Additional context**
It's completely possible that I'm not setting up the tabulator correctly.  
I've tried different options (both enabling/disabling, hasn't seemed to
make a difference.  

I'm submitting this due to illness ... sick of my manager griping about
the 'bugs' :)

**Comment by point-nine** [comment]
Yes, I've set a height on the table.  And removed the height setting on the table.  As I said I've tested with differing options and nothing helps the resizing issue.  I have a project rolling out tomorrow based on 3.5.3, I'm not comfortable updating to 4.04 until that's completed.  As you say, that's on my roadmap.

But I need to be able to roll out a stable, working version.  We can live without ipad resizing for the moment, but not desktop.  

Thank you

**Comment by point-nine** [comment]
Sir

It is not my intent to express 'anger', but 'frustration'.  I feel that
telling us to upgrade to the latest version isn't of much help.  And,
I would have appreciated you leaving the question open in case someone else
has encountered the same problems.

We will move to vers 4 when we can ... as my deadline is tomorrow, it's not
realistic to upgrade and hope for the best.  Hope is,
after all, a lousy defense against crashes/weirdness when releasing a new
app.

I'm sure you have your hands full with vers 4 upgrade and bug fixes.  But
nothing that was offered as 'help' was 'helpful'.

Just sayin ...

Thank you for your hard work!

**Comment by olifolkerd** [comment]
Hey @point-nine 

Im sorry to hear you found me closing the issue as frustrating. but please don't take that to mean i was ignoring you.

The issues section is primarily there for me to track progress on bugs and features requests, i therefore close issues when i no longer need to track them. as the touch interaction feature request, has been spoken about in previous issue, has already been requested and is already on the roadmap, where i track most updates, i did not feel it necessary to keep the issue open.

Closing the issue does not mean that other people cannot find it, it is easily searchable.

And with all due respect, releasing updates is exactly how to fix bugs and issues in the app. every time i push updates the version number increases, that is how people know that new features are in. I use the internationally recognised [Semantic Versioning](https://semver.org/) numbering system, so i have to advance the numbering as per that spec.

It is unfortunate that the functionality you need has crossed a major version threshold, but that sometimes does happen, and the release of 4.0 was well publicised for over a year.

I did not say that you should update now, that is of course entirely up to you, I was only saying that version 4.0 was available with a lot of updates over 3.5.x, it was in-fact specifically built to address a lot of user feedback from the previous version. The latest round of updates and bugfixes are on that release. just as the patch release this weekend 4.0.5 will include the next load of fixes and version 4.1.0 will include the next round of new features.

As i am the only developer that maintains this project i am afraid i simply don't have the time to go through and update all versions, bug fixes are always applied to the latest version. especially as the 4.0 update was a major internal and external rebuild making most future updates not backwards compatible.

Im sorry that you didn't find my response helpful, but im afraid there isnt a golden bullet i can give you to fix the issue with touch in your version. Integrating touch is not a simple process and requires a considerable rebuild of several parts of the system, so there is no quick fix i can offer to get it working in your version, for the time being it is just a limit of that version of Tabulator. As for the resize rows issue that was fixed in the 4.0 release.

Tabulator is always evolving, with new features being added all the time, it has 1000's of feature and I cannot anticipate everyone's needs, when Tabulator first started mobile responsiveness was not a big priority for its user base, as it has evolved it has gained functionality in this area, with responsive layout modes and basic touch events. as it progresses further new touch based functionality will be added, including the functionality you are looking for. But Tabulator has 1000's of users and i have to balance the needs of all of these users when prioritising what features and updates go in.

I am not looking to start a debate with you on this, only put across my side of the story. It is up to you to decide if Tabulator is for you.

Cheers

Oli :)
